bin/p2-bootstrap/bootstrap.go

```diff
@@ -5,7 +5,7 @@ import (
 	"os"
 	"time"
 
-	"github.com/square/p2/pkg/intent"
+	"github.com/square/p2/pkg/p2consul"
```

Comment:
My main problem with the package name is the collision with the `p2-consul` artifact we have internally. It's also redundant since it already says 'p2' in the path. Suggest renaming to `consul` or `store`.